www/grpc/blockchain_service.go

```diff
@@ -309,8 +330,35 @@ func (s *blockchainService) getBlockdetails(blockheight int64) (*pb.BlockInfo, e
 	if blockmeta == nil || block == nil {
 		return nil, fmt.Errorf("Invalid blockheight")
 	}
+	bhash := blockmeta.BlockID.Hash.Bytes()
```

Comment:
make a function for simplicity.

Reply:
never forget rules of third, make a function when you copy a piece of code for the third time .